site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -203,7 +203,7 @@ my-release-pulsar-zookeeper-0         0/1     Pending             0          16s
 
 ### 3. Enable Milvus components
 
-Milvus Operator first creates all dependencies like etcd, Pulsar, and MinIO, and then continues to create Milvus components. Therefore, you can only see the pods of etcd, Pulsar, and MinIO now.  Once all denependencies are enabled, Milvus Operator will start all Milvus components. The status of the Milvus cluster is shown as in the following output.
+Milvus Operator first creates all dependencies like etcd, Pulsar, and MinIO, and then continues to create Milvus components. Therefore, you can only see the pods of etcd, Pulsar, and MinIO now. Once all dependencies are enabled, Milvus Operator will start all Milvus components. The status of the Milvus cluster is shown in the following output.
 
 ```
 ...
```

Comment:
In line 236, please change "Milvus Pods" to "Milvus pods".

Reply:
[222fefc](https://github.com/milvus-io/milvus-docs/pull/1624/commits/222fefc2c439bf107162183df122940f9138da55)